advocacy_docs/partner_docs/VeritasGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring NetBackup for PostgreSQL'
+---
+
+
+### Prerequisites
+
+- A running EDB Postgres Advanced (EPAS)
```

Comment:
Make sure everywhere in the guide we have server at the end of product.  
EDB Postgres Advanced Server.